packages/utils/src/account.test.ts

```diff
@@ -205,5 +213,7 @@ describe('Mnemonic validation', () => {
       const password = await generateKeys(mnemonics[i], 'password')
       expect({ derivation0, derivation1, password }).toEqual(expectedPrivateKeys[i])
     }
+    // Spanish mnemonic is detected even if we pass english.
+    expect(validateMnemonic(spanishMnemonic, MnemonicLanguages.english)).toBeTruthy()
```

Comment:
As written, it would be better if this were to fail. I think we should have this be the case that if someone _does not_ specify a language, it will try any language, but if they do it should only try that one language. It's true that this may mean trying two languages more often on validation, but the computational cost is trivial.